insights/parsers/netstat.py

```diff
@@ -572,42 +614,51 @@ class SsTULPN(Parser):
     This class parse the input as a table with header:
         "Netid  State  Recv-Q  Send-Q  Local-Address-Port Peer-Address-Port  Process"
```

Comment:
You've duplicated this line after the examples so I think you need to remove this one.

Reply:
Yep, thanks, lost track of edits there.  Thanks @bfahr